src/system/ui/mediaupload/index.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import { http } from 'utils/wordpress'
+import { Icon } from 'ui'
+import { createStoreRegistry } from 'fluid/data'
```

Comment:
You don't actually need a new store registry here, but it looks like `registerStore` wasn't exported from the default store registry. I've added that to `new-ui` now so you should be able to simply `import { registerStore } from 'data'`.

Reply:
I took a pull of new-ui and merged with media app and tried to import with `import { registerStore }` but getting error.
![image](https://user-images.githubusercontent.com/50620598/73718585-f01bf880-4742-11ea-978f-692a2439d307.png)